### ISSUE TITLE
Location Detail BottomSheet

### ISSUE THREAD

**Issue by fajarnuha** [rizmaulana/kotlin-mvvm-covid19#18]
right now when we click a point location in a map, it only shows locationName, it is not so informative

instead, we can show a second bottomsheet showing the detail information about the location, reuse `partial_country_info` will do, might also add `Pin to Dashboard` button to make it easier for user to reach

**Comment by pandubaraja** [comment]
I think we can use `BottomSheetDialogFragment` class to create this feature

**Comment by fajarnuha** [comment]
i thought using `bottom_sheet_behaviour` would be just fine but only `COLLAPSE` mode, we will coordinate the behaviour with the list bottomsheet

using `BottomSheetDialogFragment` would block the the interaction with map if i recall

**Comment by pandubaraja** [comment]
Good point

> i thought using `bottom_sheet_behaviour` would be just fine but only `COLLAPSE` mode, we will coordinate the behaviour with the list bottomsheet
> 
> using `BottomSheetDialogFragment` would block the the interaction with map if i recall

Good point @fajarnuha the UX would become bad if we use `BottomSheetDialogFragment`, it will block the interaction as you mentioned.

**Comment by rizmaulana** [comment]
Good idea mas @fajarnuha @pandubaraja  I will create PR for this tonight.

**Comment by fajarnuha** [comment]
please put your name in `Assignees` if you already started doing this task @rizmaulana

**Comment by rizmaulana** [comment]
Done mas @fajarnuha

**Comment by philodi-dev** [comment]
what a about a different pincountry attached to the dashboard overview according to your localization

**Comment by fajarnuha** [comment]
> what a about a different pincountry attached to the dashboard overview according to your localization

first of all @Phil-Jayz you can create a new issue to request this feature
and yes, that would be a great idea, so please create the issue so we can discuss it

**Comment by fajarnuha** [comment]
and @rizmaulana , no need to hurry but, are you still working on this feature?

**Comment by rizmaulana** [comment]
Not yet mas @fajarnuha planning this week to working on it.

**Comment by fajarnuha** [comment]
cool, good luck 😎
